homeassistant/components/versasense/sensor.py

```diff
@@ -0,0 +1,67 @@
+"""Support for VersaSense sensor peripheral."""
+import logging
+
+from homeassistant.helpers.entity import Entity
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up the sensor platform."""
+    consumer = hass.data[DOMAIN]["consumer"]
+    peripheral = hass.data[DOMAIN][discovery_info["identifier"]]
+    parent_name = discovery_info["parent_name"]
+    unit = discovery_info["unit"]
+    measurement = discovery_info["measurement"]
+
+    async_add_entities([VSensor(peripheral, parent_name, unit, measurement, consumer)])
+
+
+class VSensor(Entity):
+    """Representation of a Sensor."""
+
+    def __init__(self, peripheral, parent_name, unit, measurement, consumer):
+        """Initialize the sensor."""
+        self._state = None
+        self._name = "{} {}".format(parent_name, measurement)
```

Comment:
We can now use f-strings instead of `str.format`, since we've updated our minimum Python version to 3.6. We prefer f-strings.